spyder/widgets/ipythonconsole/shell.py

```diff
@@ -70,6 +71,9 @@ def __init__(self, ipyclient, additional_options, interpreter_versions,
         # To save kernel replies in silent execution
         self._kernel_reply = None
 
+        color_scheme = kw['config']['JupyterWidget']['syntax_style']
+        self.set_bracket_matcher_color_scheme(color_scheme)
```

Comment:
Yes you are right and this works. I didn't see it because it is not obvious where this is inherited.